extensions/reactive-messaging-http/runtime/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-reactive-messaging-http-parent</artifactId>
+    <version>999-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>quarkus-reactive-messaging-http</artifactId>
+  <name>Quarkus - SmallRye Reactive Messaging - HTTP - Runtime</name>
+
```

Comment:
You need a description for the runtime module, it's what appears in code.quarkus.io. Try to be consistent with the rest, should start with a verb "Connect to..." or something similar.